engines/base/src/main/java/org/alfresco/transform/base/registry/TransformRegistry.java

```diff
@@ -151,44 +142,77 @@ public void handleContextRefreshedEvent(final ContextRefreshedEvent event)
      * Load the registry on application startup. This allows Components in projects that extend the t-engine base
      * to use @PostConstruct to add to {@code transformConfigSources}, before the registry is loaded.
      */
```

Comment:
Since I added IllegalStateException handling in retrieveConfig() there is no longer need for @retryable, @async annotation was needed to enable retry in asynchronous way.

Reply:
Can we still keep it `@async`?

Reply:
Unfortunately no - it was added for @Retryable, not the other way around.

Reply:
Are you sure? I am quite confident that the `@Async` does not require the `@Retryable`.

Reply:
You are right. I meant that @Retryable needed @Async. But as we discovered we can still keep @Async.